### ISSUE TITLE
Google Drive appears empty

### ISSUE THREAD

**Issue by velapage** [minbrowser/min#709]
Google Drive was working fine last week but as of one or two days ago it now appears empty. Documents I had open still work but I cannot organize anything in my drive, and trying to open folders through the dropdown menus on the left does the same— I can "open" the folder and see the path on the top of the screen but it appears empty.

Running 1.9.0 on Mac OS Mojave 10.14.3.
<img width="1150" alt="screen shot 2019-02-11 at 12 20 46 pm" src="https://user-images.githubusercontent.com/47535276/52581090-c95d0780-2df7-11e9-84f5-806f01a07071.png">

**Comment by PalmerAL** [comment]
Google Drive changed their code a few days ago that caused this to break; electron/electron#16587 has an explanation if you're interested. I added a workaround to fix this issue a few days ago in commit 4d149ad9129c9b0daea2b6d45def78abd6213835, I'll release an update in the next few days that should fix it.

In the meantime, you can get it to work again by following the instructions here to create a userscript: https://github.com/minbrowser/min/wiki/userscripts, and call it "drive.google.com.js". Place this code inside the userscript, and then restart the browser:

```
 window.chrome = {
      runtime: {
        connect: () => {
          return {
            onMessage: {
              addListener: () => {console.warn('chrome.runtime is not implemented')},
              removeListener: () => {console.warn('chrome.runtime is not implemented')},
            },
            postMessage: () => {console.warn('chrome.runtime is not implemented')},
            disconnect: () => {console.warn('chrome.runtime is not implemented')},
          }
        }
      }
    }
```

**Comment by velapage** [comment]
Great, thanks for responding so quickly!

**Comment by PalmerAL** [comment]
This is fixed in 1.9.1, you can download it from here: https://github.com/minbrowser/min/releases/tag/v1.9.1.
